src/core_plugins/vega/public/data_model/vega_parser.js

```diff
@@ -211,6 +212,37 @@ export class VegaParser {
     return result || {};
   }
 
+  _parseTooltips() {
+    if (this._config.tooltips === false) {
+      return false;
+    }
+
+    const result = this._config.tooltips || {};
+
+    if (result.position === undefined) {
```

Comment:
no need to change, but in future I'd use `typeof .... === "undefined"`, as it's more canonical and doesn't rely on obscure globals.